jetstream/analysis.py

```diff
@@ -547,6 +553,10 @@ def run(self, current_date: datetime, dry_run: bool = False) -> None:
         table_to_dataframe = dask.delayed(self.bigquery.table_to_dataframe)
 
         for period in self.config.metrics:
+            if period not in self.analysis_periods:
+                print(f"Skipping {period};")
```

Comment:
I guess we could issue it as a warning

Reply:
Ah right, bigquery only gets Warning/Error level logs, right? Well, maybe still worth just using logger instead of print so that it gets the formatting in stdout?